# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager HEAD\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2009-10-23 19:49+0000\n"
"PO-Revision-Date: 2009-10-25 09:18+0000\n"
"Last-Translator: cumulus007 <cumulus0007@gmail.com>\n"
"Language-Team: Nederlands <vertaling@vrijschrift.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2009-10-26 14:52+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:278 ../UpdateManager/Core/utils.py:278
msgid "0 KB"
msgstr "0 KB"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:281 ../UpdateManager/Core/utils.py:281
msgid "1 KB"
msgstr "1 KB"

#. TRANSLATORS: download size of small updates, e.g. "250 KB"
#: ../DistUpgrade/utils.py:284 ../UpdateManager/Core/utils.py:284
#, python-format
msgid "%.0f KB"
msgstr "%.0f KB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:287 ../UpdateManager/Core/utils.py:287
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:208 ../DistUpgrade/distro.py:423
#, python-format
msgid "Server for %s"
msgstr "Server voor %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:226 ../DistUpgrade/distro.py:232
#: ../DistUpgrade/distro.py:248
msgid "Main server"
msgstr "Hoofdserver"

#: ../DistUpgrade/distro.py:252
msgid "Custom servers"
msgstr "Andere servers"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:131
msgid "Could not calculate sources.list entry"
msgstr "Kon sources.list-vermelding niet berekenen"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:234
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Kan geen pakketbestanden vinden, mogelijk is dit geen Ubuntu-medium of bevat "
"deze niet de juiste architectuur."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:277
msgid "Failed to add the CD"
msgstr "Toevoegen van de CD is mislukt"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:278
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"De upgrade is gestopt omdat de CD niet toegevoegd kon worden. Als dit wel de "
"juiste CD is, verstuur dan een foutenrapport.\n"
"\n"
"De foutmelding was:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:130
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Verwijder pakket in slechte staat."
msgstr[1] "Verwijder pakketten in slechte staat."

#: ../DistUpgrade/DistUpgradeCache.py:133
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Het pakket '%s' verkeert in een inconsistente staat en moet opnieuw "
"geïnstalleerd worden. Er is echter geen archiefbestand gevonden voor dit "
"pakket. Wilt u verdergaan en het pakket verwijderen?"
msgstr[1] ""
"De pakketten '%s' verkeren in een inconsistente staat en moet opnieuw "
"geïnstalleerd worden. Er zijn echter geen archiefbestanden gevonden voor "
"deze pakketten. Wilt u verdergaan en de pakketten verwijderen?"

#: ../DistUpgrade/DistUpgradeCache.py:332
msgid "Broken packages"
msgstr "Niet-werkende pakketten"

#: ../DistUpgrade/DistUpgradeCache.py:333
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Uw systeem bevat niet-werkende pakketten die niet gerepareerd kunnen worden "
"met deze software. Repareer deze eerst met Synaptic of apt-get voordat u "
"verder gaat."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:609
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"Een onoplosbaar probleem deed zich voor tijdens het berekenen van de "
"upgrade:\n"
"%s\n"
"\n"
" Dit kan mogelijk veroorzaakt worden door:\n"
"\n"
" * Opwaarderen naar een pre-release versie van Ubuntu\n"
" * Het gebruik van de huidige pre-release versie van Ubuntu\n"
" * Onofficiële softwarepaketten, niet aangeleverd door Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:619
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Dit is waarschijnlijk een tijdelijk probleem.\r\n"
"Probeer het later nog eens."

#: ../DistUpgrade/DistUpgradeCache.py:622
msgid ""
"If none of this applies, then please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"Als het bovenstaande niet van toepassing is, gelieve dan deze fout te "
"rapporteren in verband met het pakket 'update-manager', en voeg de bestanden "
"uit /var/log/dist-upgrade/ als bijlage aan het rapport toe."

#: ../DistUpgrade/DistUpgradeCache.py:628
#: ../UpdateManager/UpdateManager.py:749
msgid "Could not calculate the upgrade"
msgstr "Kan de vereisten voor de upgrade niet berekenen"

#: ../DistUpgrade/DistUpgradeCache.py:678
msgid "Error authenticating some packages"
msgstr "Fout bij het bepalen van de echtheid van sommige pakketten"

#: ../DistUpgrade/DistUpgradeCache.py:679
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Het was niet mogelijk om de echtheid van sommige pakketten vast te stellen. "
"Dit kan liggen aan een tijdelijk netwerkprobleem. U kunt het later opnieuw "
"proberen. Hieronder vindt u een lijst met pakketten waarvan de echtheid niet "
"vastgesteld is."

#: ../DistUpgrade/DistUpgradeCache.py:699
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"Het pakket '%s' is gemarkeerd voor verwijdering, maar het staat op de zwarte "
"lijst voor verwijderen."

#: ../DistUpgrade/DistUpgradeCache.py:703
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Het essentiële pakket '%s' is gemarkeerd voor verwijdering."

#: ../DistUpgrade/DistUpgradeCache.py:712
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr ""
"Pakket (versie '%s') dat op de zwarte lijst staat proberen te installeren"

#: ../DistUpgrade/DistUpgradeCache.py:812
#, python-format
msgid "Can not mark '%s' for upgrade"
msgstr "Kan '%s' niet markeren voor bijwerking"

#: ../DistUpgrade/DistUpgradeCache.py:826
#, python-format
msgid "Can't install '%s'"
msgstr "Kan '%s' niet installeren"

#: ../DistUpgrade/DistUpgradeCache.py:827
msgid ""
"It was impossible to install a required package. Please report this as a "
"bug. "
msgstr ""
"Het was niet mogelijk om een vereist pakket te installeren. Gelieve dit als "
"fout te rapporteren. "

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:836
msgid "Can't guess meta-package"
msgstr "Kan het meta-pakket niet raden"

#: ../DistUpgrade/DistUpgradeCache.py:837
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"Uw systeem bevat geen van de volgende pakketten: ubuntu-desktop, kubuntu-"
"desktop of edubuntu-desktop.\n"
"Hierdoor is het niet mogelijk om uw versie van Ubuntu te bepalen.\n"
" Installeer eerst één van de bovenstaande pakketten met Synaptic of apt-get "
"voordat u verder gaat."

#: ../DistUpgrade/DistUpgradeController.py:85
msgid "Reading cache"
msgstr "Tijdelijke opslag inlezen"

#: ../DistUpgrade/DistUpgradeController.py:185
msgid "Unable to get exclusive lock"
msgstr "Kon geen exclusieve blokkering verkrijgen"

#: ../DistUpgrade/DistUpgradeController.py:186
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Waarschijnlijk draait er al een ander programma voor pakketbeheer (zoals apt-"
"get of aptitude). Sluit dit programma eerst af."

#: ../DistUpgrade/DistUpgradeController.py:232
msgid "Upgrading over remote connection not supported"
msgstr "Upgraden over een externe verbinding wordt niet ondersteund"

#: ../DistUpgrade/DistUpgradeController.py:233
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. The upgrade will abort now. Please try without "
"ssh."
msgstr ""
"U voert een upgrade uit over een externe ssh-verbinding met een frontend dat "
"dit niet ondersteunt. De upgrade wordt nu beëindigd. Probeer het opnieuw "
"zonder een ssh-verbinding."

#: ../DistUpgrade/DistUpgradeController.py:244
msgid "Continue running under SSH?"
msgstr "Doorgaan met uitvoeren via SSH?"

#: ../DistUpgrade/DistUpgradeController.py:245
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, a additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Deze sessie blijkt uitgevoerd te worden via ssh. Het is niet aan te raden "
"een upgrade uit te voeren via ssh, omdat het, in geval van problemen, "
"moeilijker is deze op te lossen.\n"
"\n"
"Als u door gaat, zal een extra ssh-service uitgevoerd worden op poort '%s'.\n"
"Wilt u doorgaan?"

#: ../DistUpgrade/DistUpgradeController.py:260
msgid "Starting additional sshd"
msgstr "Bezig met starten van extra sshd"

#: ../DistUpgrade/DistUpgradeController.py:261
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Om bij fouten een hersteloperatie te vergemakkelijken, zal een extra sshd "
"gestart worden op poort ‘%s’. Indien er iets mis gaat met de ssh die in "
"gebruik was, kunt u nog steeds verbinden met de extra sshd.\n"

#: ../DistUpgrade/DistUpgradeController.py:331
#: ../DistUpgrade/DistUpgradeController.py:376
msgid "Can not upgrade"
msgstr "Kan niet upgraden"

#: ../DistUpgrade/DistUpgradeController.py:332
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Een upgrade van '%s naar '%s' is niet mogelijk met dit programma."

#: ../DistUpgrade/DistUpgradeController.py:341
msgid "Sandbox setup failed"
msgstr "Sandbox-installatie mislukt"

#: ../DistUpgrade/DistUpgradeController.py:342
msgid "It was not possible to create the sandbox environment."
msgstr "Kon geen sandbox-omgeving opzetten."

#: ../DistUpgrade/DistUpgradeController.py:348
msgid "Sandbox mode"
msgstr "Sandbox-modus"

#: ../DistUpgrade/DistUpgradeController.py:349
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"Deze upgrade wordt uitgevoerd in de sandbox-modus (testmodus). Alle "
"wijzigingen worden geschreven naar '%s' en zullen verloren gaan bij een "
"herstart.\n"
"Er zullen *geen* veranderingen doorgevoerd worden in de systeemmappen, "
"totdat u de computer heeft herstart."

#: ../DistUpgrade/DistUpgradeController.py:377
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"Uw python-installatie is beschadigd. Repareer de symbolische link "
"‘/usr/bin/python’."

#: ../DistUpgrade/DistUpgradeController.py:403
msgid "Package 'debsig-verify' is installed"
msgstr "Pakket 'debsig-verify' is geïnstalleerd."

#: ../DistUpgrade/DistUpgradeController.py:404
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"De upgrade kan niet worden voortgezet als dat pakket geïnstalleerd is.\n"
"Verwijder het eerst met Synaptic of 'apt-get remove debsig-verify' en voer "
"de upgrade opnieuw uit."

#: ../DistUpgrade/DistUpgradeController.py:414
msgid "Include latest updates from the Internet?"
msgstr "Laatste updates van het Internet ophalen?"

#: ../DistUpgrade/DistUpgradeController.py:415
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Bij de upgrade kunnen automatisch de laatste updates van het internet "
"gedownload en geïnstalleerd worden. Als u een netwerkverbinding heeft is dit "
"ten zeerste aan te bevelen.\n"
"\n"
"Het upgrade-proces zal hierdoor langer duren, maar na afloop is uw systeem "
"geheel bijgewerkt. Als u hier niet voor kiest zult u snel na de upgrade de "
"updates moeten installeren.\n"
"Als u voor 'nee' kiest, wordt het netwerk geheel niet gebruikt."

#: ../DistUpgrade/DistUpgradeController.py:638
#, python-format
msgid "disabled on upgrade to %s"
msgstr "uitgeschakeld op upgrade naar %s"

#: ../DistUpgrade/DistUpgradeController.py:664
msgid "No valid mirror found"
msgstr "Geen geldige mirrorserver gevonden"

#: ../DistUpgrade/DistUpgradeController.py:665
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"Tijdens het doorzoeken van uw beschikbare softwarebronnen is er geen "
"spiegelserver voor de upgrade gevonden. Dit kan gebeuren als u een interne "
"spiegelserver heeft of als de spiegelserverinformatie verouderd is.\n"
"\n"
"Wilt u 'sources.list' alsnog herschrijven? Als u 'Ja' kiest, worden "
"spiegelservers '%s' tot '%s' bijgewerkt. Bij 'Nee' wordt de upgrade "
"geannuleerd."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:685
msgid "Generate default sources?"
msgstr "De standaard bronnenlijst aanmaken?"

#: ../DistUpgrade/DistUpgradeController.py:686
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Tijdens het doorzoeken van uw beschikbare softwarebronnen is er geen "
"geschikte server voor '%s' gevonden.\n"
"\n"
"Moeten de standaardservers voor '%s' toegevoegd worden? Als u 'Nee' kiest, "
"wordt de upgrade geannuleerd."

#: ../DistUpgrade/DistUpgradeController.py:721
msgid "Repository information invalid"
msgstr "De informatie over de pakketbronnen is ongeldig"

#: ../DistUpgrade/DistUpgradeController.py:722
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug."
msgstr ""
"Het upgraden van de informatie over de pakketbronnen heeft het bestand "
"ongeldig gemaakt. Rapporteer dit als een fout."

#: ../DistUpgrade/DistUpgradeController.py:728
msgid "Third party sources disabled"
msgstr "Pakketbronnen van derden uitgeschakeld"

#: ../DistUpgrade/DistUpgradeController.py:729
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Enkele pakketbronnen van derden in uw sources.list zijn gedeactiveerd. U "
"kunt ze na de upgrade weer activeren met het programma ‘software-properties’ "
"of met ‘Synaptic’."

#: ../DistUpgrade/DistUpgradeController.py:769
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pakket in onsamenhangende staat"
msgstr[1] "Pakketen in onsamenhangende staat"

#: ../DistUpgrade/DistUpgradeController.py:772
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"De pakketten '%s' verkeren in een inconsistente staat en moet opnieuw "
"geïnstalleerd worden. Er zijn echter geen archiefbestanden gevonden voor "
"deze pakketten. Gelieve het pakket handmatig te herinstalleren of te "
"verwijderen uit het systeem."
msgstr[1] ""
"De pakketten '%s' verkeren in een inconsistente staat en moet opnieuw "
"geïnstalleerd worden. Er zijn echter geen archiefbestanden gevonden voor "
"deze pakketten. Gelieve de pakketten handmatig te herinstalleren of te "
"verwijderen uit het systeem."

#: ../DistUpgrade/DistUpgradeController.py:820
msgid "Error during update"
msgstr "Fout tijdens het updaten"

#: ../DistUpgrade/DistUpgradeController.py:821
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Tijdens de update is er iets misgegaan. De oorzaak is meestal een "
"netwerkprobleem. Controleer uw netwerkverbinding en probeer het opnieuw."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:830
#: ../UpdateManager/UpdateManager.py:572
msgid "Not enough free disk space"
msgstr "Niet genoeg vrije schijfruimte"

#: ../DistUpgrade/DistUpgradeController.py:831
#, python-format
msgid ""
"The upgrade is now aborted. The upgrade needs a total of %s free space on "
"disk '%s'. Please free at least an additional %s of disk space on '%s'. "
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"De upgrade is onderbroken. Er is een totaal van %s vrije ruimte nodig op "
"schijf '%s'. Maak tenminste %s ruimte vrij op schijf '%s'. Leeg uw "
"prullenbak, en verwijder pakketten van eerdere installaties met 'sudo apt-"
"get clean'."

#: ../DistUpgrade/DistUpgradeController.py:871
msgid "Support for some applications ended"
msgstr "Ondersteuning voor bepaalde toepassingen is beëindigd."

#: ../DistUpgrade/DistUpgradeController.py:872
msgid ""
"Canonical Ltd. no longer provides support for the following software "
"packages. You can still get support from the community.\n"
"\n"
"If you have not enabled community maintained software (universe), these "
"packages will be suggested for removal at the end of the upgrade."
msgstr ""
"De volgende pakketten worden niet meer door Canonical Ltd. ondersteund. U "
"kunt nog wel ondersteuning krijgen uit de gemeenschap.\n"
"\n"
"Indien ‘door de gemeenschap onderhouden software’ (universe) niet "
"geactiveerd is, zal bij de volgende stap gevraagd worden om deze pakketten "
"te verwijderen."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:881
#: ../DistUpgrade/DistUpgradeController.py:1551
msgid "Calculating the changes"
msgstr "Wijzigingen worden doorgerekend"

#. ask the user if he wants to do the changes
#: ../DistUpgrade/DistUpgradeController.py:896
msgid "Do you want to start the upgrade?"
msgstr "Wilt u beginnen met de upgrade?"

#: ../DistUpgrade/DistUpgradeController.py:952
msgid "Upgrade canceled"
msgstr "Upgrade geannuleerd"

#: ../DistUpgrade/DistUpgradeController.py:953
msgid ""
"The upgrade cancels now and the original system state is restored. You can "
"resume the upgrade at a later time."
msgstr ""
"De upgrade wordt nu geannuleerd en de originele systeemstatus is hersteld. U "
"kunt de upgrade op een later tijdstip voortzetten."

#: ../DistUpgrade/DistUpgradeController.py:958
#: ../DistUpgrade/DistUpgradeController.py:1045
msgid "Could not download the upgrades"
msgstr "Kon de upgrades niet downloaden"

#: ../DistUpgrade/DistUpgradeController.py:959
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far are kept."
msgstr ""
"De upgrade is onderbroken. Controleer uw internetverbinding of "
"installatiemedium, en probeer opnieuw. Reeds gedownloade bestanden blijven "
"bewaard."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1012
#: ../DistUpgrade/DistUpgradeController.py:1135
msgid "Error during commit"
msgstr "Fout bij het toepassen"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1014
#: ../DistUpgrade/DistUpgradeController.py:1174
msgid "Restoring original system state"
msgstr "De oorspronkelijke toestand wordt hersteld"

#: ../DistUpgrade/DistUpgradeController.py:1015
#: ../DistUpgrade/DistUpgradeController.py:1029
msgid "Could not install the upgrades"
msgstr "Kon de upgrades niet installeren"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1020
msgid ""
"The upgrade is now aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"De upgrade is onderbroken. Het systeem is mogelijk in een onstabiele "
"situatie. Een herstelprocedure wordt nu uitgevoerd (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1025
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Gelieve deze fout te rapporteren in verband met het pakket 'update-manager', "
"en voeg de bestanden uit /var/log/dist-upgrade/ als bijlage aan de "
"rapportering toe.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1046
msgid ""
"The upgrade is now aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"De upgrade is onderbroken. Controleer uw internetverbinding of "
"installatiemedium, en probeer opnieuw. "

#: ../DistUpgrade/DistUpgradeController.py:1123
msgid "Remove obsolete packages?"
msgstr "Overbodige pakketten verwijderen?"

#: ../DistUpgrade/DistUpgradeController.py:1124
#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Keep"
msgstr "_Behouden"

#: ../DistUpgrade/DistUpgradeController.py:1124
msgid "_Remove"
msgstr "_Verwijderen"

#: ../DistUpgrade/DistUpgradeController.py:1136
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Tijdens het opruimen deed zich een probleem voor. Zie onderstaand bericht "
"voor meer informatie. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1212
msgid "Required depends is not installed"
msgstr "De vereiste afhankelijkheden zijn niet geïnstalleerd"

#: ../DistUpgrade/DistUpgradeController.py:1213
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Het vereiste pakket ‘%s’ is niet geïnstalleerd. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1458
#: ../DistUpgrade/DistUpgradeController.py:1525
msgid "Checking package manager"
msgstr "Pakketbeheer controleren"

#: ../DistUpgrade/DistUpgradeController.py:1463
msgid "Preparing the upgrade failed"
msgstr "Voorbereiden van de upgrade is mislukt"

#: ../DistUpgrade/DistUpgradeController.py:1464
msgid ""
"Preparing the system for the upgrade failed. Please report this as a bug "
"against the 'update-manager' package and include the files in /var/log/dist-"
"upgrade/ in the bug report."
msgstr ""
"De voorbereidingen voor de upgrade zijn mislukt. Gelieve deze fout te "
"rapporteren in verband met het pakket 'update-manager', en voeg de bestanden "
"uit /var/log/dist-upgrade/ als bijlage aan de rapportering toe."

#: ../DistUpgrade/DistUpgradeController.py:1480
msgid "Getting upgrade prerequisites failed"
msgstr "Het voorbereiden van de upgrade is mislukt"

#: ../DistUpgrade/DistUpgradeController.py:1481
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug against the 'update-manager' package and include "
"the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""
"Het systeem was niet in staat de voorwaarden voor de upgrade te achterhalen. "
"De upgrade wordt nu afgebroken en het systeem wordt hersteld naar de "
"originele staat.\n"
"\n"
"Gelieve deze fout te rapporteren in verband met het pakket 'update-manager', "
"en voeg de bestanden uit /var/log/dist-upgrade/ als bijlage aan de "
"rapportering toe."

#: ../DistUpgrade/DistUpgradeController.py:1511
msgid "Updating repository information"
msgstr "Updaten van de informatie over de pakketbronnen"

#: ../DistUpgrade/DistUpgradeController.py:1539
msgid "Invalid package information"
msgstr "Ongeldige pakketinformatie"

#: ../DistUpgrade/DistUpgradeController.py:1540
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"Nadat uw pakketinformatie werd bijgewerkt, kon het essentiële pakket '%s' "
"niet meer gevonden worden.\n"
"Dit is een indicatie van een ernstige fout. Gelieve deze fout te rapporteren "
"in verband met het pakket 'update-manager', en voeg de bestanden uit "
"/var/log/dist-upgrade/ als bijlage aan de rapportering toe."

#: ../DistUpgrade/DistUpgradeController.py:1557
#: ../DistUpgrade/DistUpgradeController.py:1605
msgid "Fetching"
msgstr "Ophalen"

#: ../DistUpgrade/DistUpgradeController.py:1563
#: ../DistUpgrade/DistUpgradeController.py:1609
msgid "Upgrading"
msgstr "Bezig met upgraden"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1568
#: ../DistUpgrade/DistUpgradeController.py:1611
#: ../DistUpgrade/DistUpgradeController.py:1618
#: ../DistUpgrade/DistUpgradeController.py:1623
msgid "Upgrade complete"
msgstr "Upgrade is voltooid"

#: ../DistUpgrade/DistUpgradeController.py:1569
#: ../DistUpgrade/DistUpgradeController.py:1612
#: ../DistUpgrade/DistUpgradeController.py:1619
msgid ""
"The upgrade is completed but there were errors during the upgrade process."
msgstr ""
"De upgrade is voltooid, maar zijn fouten opgetreden tijdens het "
"upgradeproces."

#: ../DistUpgrade/DistUpgradeController.py:1576
msgid "Searching for obsolete software"
msgstr "Zoeken naar overbodige software"

#: ../DistUpgrade/DistUpgradeController.py:1581
msgid "System upgrade is complete."
msgstr "Systeemupgrade is voltooid."

#: ../DistUpgrade/DistUpgradeController.py:1624
msgid "The partial upgrade was completed."
msgstr "De gedeeltelijke upgrade is voltooid."

#: ../DistUpgrade/DistUpgradeQuirks.py:149
msgid "No ARMv6 CPU"
msgstr "Geen ARMv6-CPU"

#: ../DistUpgrade/DistUpgradeQuirks.py:150
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Uw systeem gebruikt een ARM-CPU ouder dan de ARMv6-architectuur. Alle "
"pakketten in karmic zijn gecompileerd voor ARMv6 als de minimale "
"architectuur. Het is niet mogelijk om uw systeem naar een nieuwe Ubuntu-"
"versie te upgraden met deze hardware."

#: ../DistUpgrade/DistUpgradeQuirks.py:164
msgid "No init available"
msgstr "Geen init beschikbaar"

#: ../DistUpgrade/DistUpgradeQuirks.py:165
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 9.10 cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Het lijkt erop dat uw systeem gevirtualiseerd is zonder een init-daemon, "
"bijvoorbeeld Linux-VServer. Ubuntu 9.10 kan niet functioneren in dit type "
"virtualisatie. U zult eerst de instellingen van uw virtuele machine moeten "
"aanpassen.\n"
"\n"
"Weet u zeker dat u door wilt gaan?"

#. TRANSLATORS: this string is not displayed in the 9.10 upgrade
#. (but it will be used for 8.04 -> 10.04 LTS upgrades)
#: ../DistUpgrade/DistUpgradeQuirks.py:186
#: ../DistUpgrade/DistUpgradeQuirks.py:271
#: ../DistUpgrade/DistUpgradeQuirks.py:298
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Mogelijk worden tijdens een upgrade de bureaubladeffecten minder, en worden "
"de prestaties bij games en andere grafisch intensieve programma's negatief "
"beïnvloed."

#: ../DistUpgrade/DistUpgradeQuirks.py:190
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Deze computer gebruikt momenteel het grafisch stuurprogramma 'fglrx' van "
"AMD. Voor Ubuntu 10.04 is geen stuurprogramma beschikbaar dat met uw "
"hardware werkt.\n"
"Wilt u doorgaan?"

#: ../DistUpgrade/DistUpgradeQuirks.py:275
#: ../DistUpgrade/DistUpgradeQuirks.py:302
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Deze computer gebruikt momenteel het grafisch stuurprogramma 'nvidia' van "
"NVIDIA. Voor Ubuntu 10.04 is geen stuurprogramma beschikbaar dat met uw "
"hardware werkt.\n"
"Wilt u doorgaan?"

#: ../DistUpgrade/DistUpgradeQuirks.py:472
msgid "evms in use"
msgstr "evms wordt gebruikt"

#: ../DistUpgrade/DistUpgradeQuirks.py:473
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"Uw systeem gebruikt 'evms' volumebeheer in /proc/mounts. De software 'evms' "
"wordt niet langer ondersteund, schakel het uit en voer de upgrade nogmaals "
"uit wanneer dit gebeurd is."

#: ../DistUpgrade/DistUpgradeMain.py:43
msgid "Sandbox upgrade using aufs"
msgstr "Sandbox upgrade via aufs"

#: ../DistUpgrade/DistUpgradeMain.py:45
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Gebruik het opgegeven pad om te zoeken voor een cdrom met pakketen die "
"geupgrade kunnen worden"

#: ../DistUpgrade/DistUpgradeMain.py:51
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Te gebruiken frontend. Momenteel beschikbaar: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:54
msgid "*DEPRECATED* this option will be ignore"
msgstr "*Verouderd* deze optie zal worden genegeerd"

#: ../DistUpgrade/DistUpgradeMain.py:57
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Een gedeeltelijke upgrade uitvoeren (sources.list wordt niet herschreven)"

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Set datadir"
msgstr "Datamap instellen"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:106
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Plaats '%s' in station '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:203
msgid "Fetching is complete"
msgstr "Het downloaden is voltooid"

#: ../DistUpgrade/DistUpgradeViewGtk.py:137
#: ../DistUpgrade/DistUpgradeViewKDE.py:216
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Ophalen bestand %li van %li met %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:138
#: ../DistUpgrade/DistUpgradeViewGtk.py:273
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#: ../DistUpgrade/DistUpgradeViewKDE.py:357
#, python-format
msgid "About %s remaining"
msgstr "Ongeveer %s resterend"

#: ../DistUpgrade/DistUpgradeViewGtk.py:140
#: ../DistUpgrade/DistUpgradeViewKDE.py:219
#, python-format
msgid "Fetching file %li of %li"
msgstr "Downloaden van bestand %li van %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:172
#: ../DistUpgrade/DistUpgradeViewKDE.py:253
msgid "Applying changes"
msgstr "Wijzigingen worden doorgevoerd"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:197
#: ../DistUpgrade/DistUpgradeViewKDE.py:266
msgid "dependency problems - leaving unconfigured"
msgstr "vereistenproblemen - blijft ongeconfigureerd"

#: ../DistUpgrade/DistUpgradeViewGtk.py:202
#: ../DistUpgrade/DistUpgradeViewKDE.py:268
#, python-format
msgid "Could not install '%s'"
msgstr "Kon '%s' niet installeren"

#: ../DistUpgrade/DistUpgradeViewGtk.py:203
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"De upgrade zal doorgaan, maar het pakket '%s' zal mogelijk niet werken. "
"Overweeg er een foutrapport over in te dienen."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:219
#: ../DistUpgrade/DistUpgradeViewKDE.py:290
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Het aangepaste configuratiebestand vervangen?\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeViewGtk.py:220
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Wanneer u besluit dit configuratiebestand te vervangen door een nieuwere "
"versie zullen al uw gemaakte wijzigingen verloren gaan."

#: ../DistUpgrade/DistUpgradeViewGtk.py:233
#: ../DistUpgrade/DistUpgradeViewKDE.py:309
msgid "The 'diff' command was not found"
msgstr "Het commando 'diff' is niet gevonden"

#: ../DistUpgrade/DistUpgradeViewGtk.py:419
#: ../DistUpgrade/DistUpgradeViewText.py:107
msgid "A fatal error occurred"
msgstr "Er is een ernstige fout opgetreden"

#: ../DistUpgrade/DistUpgradeViewGtk.py:420
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Gelieve deze fout te rapporteren (als u dat nog niet gedaan heeft) in "
"verband met het pakket 'update-manager', en voeg de bestanden /var/log/dist-"
"upgrade/main.log en /var/log/dist-upgrade/apt.log als bijlage aan de "
"rapportering toe. De upgrade is nu afgebroken.\n"
"Uw originele pakkettenlijst (sources.list) is opgeslagen in "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk.py:435
msgid "Ctrl-c pressed"
msgstr "Ctrl-c ingedrukt"

#: ../DistUpgrade/DistUpgradeViewGtk.py:436
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Dit zal de taak onderbreken waardoor uw systeem mogelijk niet meer correct "
"werkt. Weet u zeker dat u dit wilt doen?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:581
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Sluit alle openstaande toepassingen en documenten om dataverlies te "
"voorkomen."

#: ../DistUpgrade/DistUpgradeViewGtk.py:595
#, python-format
msgid "<b>Downgrade %s</b>"
msgstr "<b>%s downgraden</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:597
#: ../DistUpgrade/DistUpgradeViewKDE.py:757
#, python-format
msgid "Remove %s"
msgstr "Verwijder %s"

#: ../DistUpgrade/DistUpgradeViewGtk.py:602
#: ../DistUpgrade/DistUpgradeViewKDE.py:759
#, python-format
msgid "Install %s"
msgstr "%s installeren"

#: ../DistUpgrade/DistUpgradeViewGtk.py:604
#: ../DistUpgrade/DistUpgradeViewKDE.py:761
#, python-format
msgid "Upgrade %s"
msgstr "%s upgraden"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:191
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Media Change"
msgstr "Medium wisselen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:321
msgid "Show Difference >>>"
msgstr "Verschillen tonen >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:324
msgid "<<< Hide Difference"
msgstr "<<< Verschillen verbergen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:538
msgid "Error"
msgstr "Foutmelding"

#: ../DistUpgrade/DistUpgradeViewKDE.py:550
msgid "&Cancel"
msgstr "&Annuleren"

#: ../DistUpgrade/DistUpgradeViewKDE.py:552
#: ../DistUpgrade/DistUpgradeViewKDE.py:789
msgid "&Close"
msgstr "Sl&uiten"

#: ../DistUpgrade/DistUpgradeViewKDE.py:598
msgid "Show Terminal >>>"
msgstr "Terminalvenster weergeven >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:601
msgid "<<< Hide Terminal"
msgstr "<<< Terminalvenster verbergen"

#: ../DistUpgrade/DistUpgradeViewKDE.py:681
msgid "Information"
msgstr "Informatie"

#: ../DistUpgrade/DistUpgradeViewKDE.py:731
#: ../DistUpgrade/DistUpgradeViewKDE.py:772
#: ../DistUpgrade/DistUpgradeViewKDE.py:775 ../DistUpgrade/DistUpgrade.ui.h:9
msgid "Details"
msgstr "Details"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "Restart required"
msgstr "De computer moet opnieuw opgestart worden"

#: ../DistUpgrade/DistUpgradeViewKDE.py:785 ../DistUpgrade/DistUpgrade.ui.h:5
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Herstart het systeem om de upgrade te voltooien</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:788 ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Restart Now"
msgstr "_Nu herstarten"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:800
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>De upgrade annuleren?</big></b>\n"
"\n"
"Het systeem kan instabiel worden als u de upgrade annuleert. U wordt sterk "
"geadviseerd om met de upgrade door te gaan."

#: ../DistUpgrade/DistUpgradeViewKDE.py:804
msgid "Cancel Upgrade?"
msgstr "Upgrade annuleren?"

#: ../DistUpgrade/DistUpgradeView.py:60
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dag"
msgstr[1] "%li dagen"

#: ../DistUpgrade/DistUpgradeView.py:62
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li uur"
msgstr[1] "%li uur"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minuut"
msgstr[1] "%li minuten"

#: ../DistUpgrade/DistUpgradeView.py:65
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li seconde"
msgstr[1] "%li seconden"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:78
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:93
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:114
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Deze download zal ongeveer %s duren met een 1 Mbit DSL-verbinding en "
"ongeveer %s duren met een 56k-modem."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:118
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Deze download duurt met uw verbinding ongeveer %s. "

#: ../DistUpgrade/DistUpgradeView.py:212 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Preparing to upgrade"
msgstr "Upgrade voorbereiden"

#: ../DistUpgrade/DistUpgradeView.py:213
msgid "Getting new software channels"
msgstr "Nieuwe softwarekanalen worden opgehaald"

#: ../DistUpgrade/DistUpgradeView.py:214 ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Getting new packages"
msgstr "Ophalen van nieuwe pakketten"

#: ../DistUpgrade/DistUpgradeView.py:215 ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Installing the upgrades"
msgstr "Installeren van de upgrades"

#: ../DistUpgrade/DistUpgradeView.py:216 ../DistUpgrade/DistUpgrade.ui.h:8
msgid "Cleaning up"
msgstr "Opruimen"

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:289
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d pakket zal verwijderd worden."
msgstr[1] "%d pakketten zullen verwijderd worden."

#: ../DistUpgrade/DistUpgradeView.py:294
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d nieuw pakket zal geïnstalleerd worden."
msgstr[1] "%d nieuwe pakketten zullen geïnstalleerd worden."

#: ../DistUpgrade/DistUpgradeView.py:300
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d pakket zal een upgrade krijgen"
msgstr[1] "%d pakketten zullen een upgrade krijgen."

#: ../DistUpgrade/DistUpgradeView.py:305
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"U moet in totaal %s downloaden. "

#: ../DistUpgrade/DistUpgradeView.py:309
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr ""
"Het upgraden kan enkele uren in beslag nemen en kan tussentijds niet worden "
"afgebroken."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:315 ../UpdateManager/UpdateManager.py:502
msgid "Your system is up-to-date"
msgstr "Uw systeem is up-to-date"

#: ../DistUpgrade/DistUpgradeView.py:316
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Er zijn geen upgrades beschikbaar voor uw systeem. De upgrade wordt nu "
"afgebroken."

#: ../DistUpgrade/DistUpgradeView.py:329
msgid "Reboot required"
msgstr "De computer moet opnieuw opgestart worden"

#: ../DistUpgrade/DistUpgradeView.py:330
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Het upgraden is voltooid en herstarten is noodzakelijk. Wilt u dit nu doen?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:134
#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:134
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
msgid "Could not run the upgrade tool"
msgstr "Kon het upgradeprogramma niet uitvoeren"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug"
msgstr ""
"Dit is waarschijnlijk een fout in het upgrade-programma. Rapporteer deze "
"fout."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool signature"
msgstr "Handtekening upgradeprogramma"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:207
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:207
msgid "Upgrade tool"
msgstr "Upgradeprogramma"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:237
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:237
msgid "Failed to fetch"
msgstr "Ophalen is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:238
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Het ophalen van de upgrade is mislukt. Er is mogelijk een netwerkprobleem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:242
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:242
msgid "Failed to extract"
msgstr "Uitpakken is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:243
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Het uitpakken van de upgrade is mislukt. Er is mogelijk een probleem met het "
"netwerk of de server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Verification failed"
msgstr "Verificatie is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Het verifiëren van de upgrade is mislukt. Er is mogelijk een probleem met "
"het netwerk of de server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid "Authentication failed"
msgstr "Echtheidscontrole is mislukt"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:255
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"De echtheidscontrole van de upgrade is mislukt. Er is mogelijk een probleem "
"met het netwerk of de server. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:263
#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:263
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
msgid "Can not run the upgrade"
msgstr "Kan de upgrade niet uitvoeren"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:264
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:264
msgid ""
"This usually is caused by a system were /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Dit wordt meestal veroorzaakt door een system waar /tmp aangekoppeld is als "
"‘noexec’. Koppel de partitie opnieuw aan zonder ‘noexec’ en voer de upgrade "
"opnieuw uit."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:270
#, python-format
msgid "The error message is '%s'."
msgstr "De foutmelding is ‘%s’."

#: ../DistUpgrade/DistUpgradeViewText.py:108
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade is now aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Gelieve deze fout te rapporteren in verband met het pakket 'update-manager', "
"en voeg de bestanden /var/log/dist-upgrade/main.log en /var/log/dist-"
"upgrade/apt.log als bijlage aan de rapportering toe. De upgrade is nu "
"afgebroken.\n"
"Uw originele pakkettenlijst (sources.list) is opgeslagen in "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:132
msgid "Aborting"
msgstr "Afbreken"

#: ../DistUpgrade/DistUpgradeViewText.py:137
msgid "Demoted:\n"
msgstr "Gedegradeerd:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:191
#: ../DistUpgrade/DistUpgradeViewText.py:198
msgid "Continue [yN] "
msgstr "_Doorgaan [jN] "

#: ../DistUpgrade/DistUpgradeViewText.py:169
#: ../DistUpgrade/DistUpgradeViewText.py:191
msgid "Details [d]"
msgstr "Details [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:173
#: ../DistUpgrade/DistUpgradeViewText.py:201
msgid "y"
msgstr "j"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:176
#: ../DistUpgrade/DistUpgradeViewText.py:208
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:179
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:183
#, python-format
msgid "Remove: %s\n"
msgstr "Verwijderen: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:186
#, python-format
msgid "Install: %s\n"
msgstr "Installeren: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:189
#, python-format
msgid "Upgrade: %s\n"
msgstr "Upgraden: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:205
msgid "Continue [Yn] "
msgstr "Doorgaan [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:226
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Om de upgrade te voltooien moet de computer opnieuw worden opgestart\n"
"Wanneer u 'j' selecteert zal de computer herstart worden."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Actieve upgrade afbreken?</big></b>\n"
"\n"
"Het systeem kan zich in een onbruikbare staat bevinden als u de upgrade "
"afbreekt. U wordt sterk aangeraden om door te gaan met de upgrade."

#: ../DistUpgrade/DistUpgrade.ui.h:6
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Upgrade starten?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:7
msgid "<b><big>Upgrading Ubuntu to version 9.10</big></b>"
msgstr "<b><big>Ubuntu opwaarderen naar versie 9.10</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Verschillen tussen de bestanden"

#: ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Distribution Upgrade"
msgstr "Distributieupgrade"

#: ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Restarting the computer"
msgstr "Computer herstarten"

#: ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Setting new software channels"
msgstr "Instellen van de nieuwe softwarekanalen"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Terminal"
msgstr "Terminalvenster"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "_Cancel Upgrade"
msgstr "Upgrade _annuleren"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "_Continue"
msgstr "_Doorgaan"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Replace"
msgstr "_Vervangen"

#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Report Bug"
msgstr "_Fout rapporteren"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Resume Upgrade"
msgstr "_Upgrade hervatten"

#: ../DistUpgrade/DistUpgrade.ui.h:25
msgid "_Start Upgrade"
msgstr "Upgrade _starten"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:42
msgid "Please wait, this can take some time."
msgstr "Een ogenblik geduld, dit kan even duren."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:44
msgid "Update is complete"
msgstr "De update is voltooid"

#: ../UpdateManager/DistUpgradeFetcher.py:85
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "Could not find the release notes"
msgstr "Kon de versie-informatie niet vinden"

#: ../UpdateManager/DistUpgradeFetcher.py:86
#: ../UpdateManager/DistUpgradeFetcherKDE.py:104
msgid "The server may be overloaded. "
msgstr "De server is waarschijnlijk overbelast. "

#: ../UpdateManager/DistUpgradeFetcher.py:96
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Could not download the release notes"
msgstr "Kon de versie-informatie niet downloaden"

#: ../UpdateManager/DistUpgradeFetcher.py:97
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Please check your internet connection."
msgstr "Controleer uw internetverbinding."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:62
#: ../UpdateManager/DistUpgradeFetcherKDE.py:85
msgid "Upgrade"
msgstr "Upgraden"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:89
#: ../data/glade/UpdateManager.ui.h:12
msgid "Release Notes"
msgstr "Versie-informatie"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:128
#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
msgid "Downloading additional package files..."
msgstr "Aanvullende pakketbestanden worden gedownload..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Bestand %s van %s bij %sB/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
#, python-format
msgid "File %s of %s"
msgstr "Bestand %s van %s"

#: ../UpdateManager/ChangelogViewer.py:70
msgid "Open Link in Browser"
msgstr "Link in browser openen"

#: ../UpdateManager/ChangelogViewer.py:72
msgid "Copy Link to Clipboard"
msgstr "Link naar klembord kopiëren"

#: ../UpdateManager/GtkProgress.py:115
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Downloaden van bestand %(current)li van %(total)li met %(speed)s/s"

#: ../UpdateManager/GtkProgress.py:120
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Downloaden van bestand %(current)li van %(total)li"

#: ../UpdateManager/UpdateManager.py:303
#, python-format
msgid "Version %s: \n"
msgstr "Versie %s: \n"

#: ../UpdateManager/UpdateManager.py:360
msgid "Downloading list of changes..."
msgstr "Een overzicht van de wijzigingen wordt gedownload..."

#: ../UpdateManager/UpdateManager.py:400
msgid "_Uncheck All"
msgstr "Alles _deselecteren"

#: ../UpdateManager/UpdateManager.py:406
msgid "_Check All"
msgstr "Alles _controleren"

#: ../UpdateManager/UpdateManager.py:450
#, python-format
msgid "%s selected. Download size: %s"
msgstr "%s geselecteerd. Downloadgrootte: %s"

#: ../UpdateManager/UpdateManager.py:455
msgid "Unknown download size"
msgstr "Downloadgrootte onbekend"

#: ../UpdateManager/UpdateManager.py:463
msgid ""
"It is unknown when the package information was updated last. Please try "
"clicking on the 'Check' button to update the information."
msgstr ""
"Het is niet bekend wanneer de pakketinformatie voor het laatst is "
"bijgewerkt. Klik op de knop 'Controleren' om de informatie bij te werken."

#: ../UpdateManager/UpdateManager.py:472
#, python-format
msgid "The package information was last updated %s day ago."
msgid_plural "The package information was last updated %s days ago."
msgstr[0] "De pakketinformatie is %s dag geleden voor het laatst bijgewerkt."
msgstr[1] ""
"De pakketinformatie is %s dagen geleden voor het laatst bijgewerkt."

#: ../UpdateManager/UpdateManager.py:476
#, python-format
msgid "The package information was last updated %s hour ago."
msgid_plural "The package information was last updated %s hours ago."
msgstr[0] "De pakketinformatie is %s uur geleden voor het laatst bijgewerkt."
msgstr[1] "De pakketinformatie is %s uur geleden voor het laatst bijgewerkt."

#: ../UpdateManager/UpdateManager.py:480
msgid "The package information was last updated less than one hour ago."
msgstr ""
"De pakketinformatie is minder dan een uur geleden voor het laatst bijgewerkt."

#: ../UpdateManager/UpdateManager.py:500 ../data/glade/UpdateManager.ui.h:22
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr ""
"Software-updates herstellen fouten, verhelpen veiligheidsproblemen en "
"leveren nieuwe mogelijkheden."

#: ../UpdateManager/UpdateManager.py:519
msgid "Welcome to Ubuntu"
msgstr "Welkom bij Ubuntu"

#: ../UpdateManager/UpdateManager.py:520
msgid ""
"These software updates have been issued since Ubuntu was released. If you "
"don't want to install them now, choose \"Update Manager\" from the "
"Administration Menu later."
msgstr ""
"Deze softwareupdates zijn vrijgegeven sinds Ubuntu is uitgegeven. Als u ze "
"op een later tijdstip wilt installeren, kies dan \"Updatebeheerder\" uit het "
"menu Beheer."

#: ../UpdateManager/UpdateManager.py:523
msgid "Software updates are available for this computer"
msgstr "Er zijn softwareupdates beschikbaar voor deze computer"

#: ../UpdateManager/UpdateManager.py:524
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr ""
"Als u deze updates later wilt installeren, kies dan \"Updatebeheer\" uit het "
"menu Beheer."

#: ../UpdateManager/UpdateManager.py:573
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"De update vereist %s vrije ruimte op schijf '%s'. Maak tenminste %s ruimte "
"vrij op '%s'. leeg uw prullenbak, en verwijder tijdelijke pakketbestanden "
"van vorige installaties met 'sudo apt-get clean'."

#: ../UpdateManager/UpdateManager.py:610
#: ../UpdateManagerHildon/UpdateManagerHildon.py:114
msgid "Reading package information"
msgstr "Pakketinformatie lezen"

#: ../UpdateManager/UpdateManager.py:727
msgid "Could not initialize the package information"
msgstr "Kon de pakketinformatie niet initialiseren"

#: ../UpdateManager/UpdateManager.py:728
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"Er is een onoplosbaar probleem geconstateerd bij het initialiseren van de "
"pakketinformatie.\n"
"\n"
"Rapporteer deze fout bij het pakket ‘update-manager’ en voeg de volgende "
"foutmelding bij:\n"

#: ../UpdateManager/UpdateManager.py:750
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"Er is een onoplosbaar probleem geconstateerd bij het doorrekenen van de "
"upgrade.\n"
"\n"
"Rapporteer deze fout bij het pakket ‘update-manager’ en voeg de volgende "
"foutmelding bij:"

#: ../UpdateManager/UpdateManager.py:773
msgid " (New install)"
msgstr " (Nieuwe installatie)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:777
#, python-format
msgid "(Size: %s)"
msgstr "(Grootte: %s)"

#: ../UpdateManager/UpdateManager.py:779
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "Van versie: %(old_version)s naar %(new_version)s"

#: ../UpdateManager/UpdateManager.py:783
#, python-format
msgid "Version %s"
msgstr "Versie %s"

#: ../UpdateManager/UpdateManager.py:796
msgid "Your Ubuntu release is not supported anymore"
msgstr "Uw versie van Ubuntu wordt nier meer ondersteund"

#: ../UpdateManager/UpdateManager.py:797
msgid ""
"You will not get any further security fixes or critical updates. Please "
"Upgrade to a later version of Ubuntu Linux."
msgstr ""
"U zult geen veiligheidsupdates of andere belangrijke updates meer ontvangen. "
"Waardeer uw Ubuntu Linux op naar een latere versie."

#: ../UpdateManager/UpdateManager.py:804
msgid "Upgrade information"
msgstr "Upgrade-informatie"

#: ../UpdateManager/UpdateManager.py:831
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>Nieuwe Ubuntu-versie '%s' is beschikbaar</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:870
msgid "Software index is broken"
msgstr "Software-index is beschadigd"

#: ../UpdateManager/UpdateManager.py:871
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"Het is nu niet mogelijk om software te installeren of te verwijderen. "
"Gebruik het pakkettenbeheerprogramma \"Synaptic\" of gebruik \"sudo apt-get "
"install -f\" in een terminalvenster om eerst dit probleem te verhelpen."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:136
msgid "Downloading Package Information"
msgstr "Ophalen van pakketinformatie"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:137
msgid ""
"The repositories are being checked for new, removed, or updated software "
"packages"
msgstr ""
"Er wordt gekeken of er nieuwe, verwijderde, of bijgewerkte softwarepakketten "
"beschikbaar zijn"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:145
msgid "Downloading Package Updates"
msgstr "Pakketupdates downloaden"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:146
msgid ""
"The selected package updates are being downloaded and installed on the system"
msgstr ""
"De geselecteerde pakketupdates worden gedownload en geïnstalleerd op het "
"systeem"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:152
msgid "run_synaptic called with unknown action"
msgstr "run-synaptic opgeroepen met onbekende actie"

#: ../UpdateManagerText/UpdateManagerText.py:26
msgid "Cancel"
msgstr "Annuleren"

#: ../UpdateManagerText/UpdateManagerText.py:27
msgid "Install"
msgstr "Installeren"

#: ../UpdateManagerText/UpdateManagerText.py:45
msgid "Building Updates List"
msgstr "Updateslijst genereren"

#: ../UpdateManagerText/UpdateManagerText.py:48
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"Er kon geen normale upgrade berekend worden, voer uit: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"Dit kan veroorzaakt worden door:\n"
" * Een vorige update die niet voltooid werd\n"
" * Problemen met de geïnstalleerde software\n"
" * Onofficiële softwarepakketten die niet door Ubuntu geleverd worden\n"
" * Normale veranderingen van een pre-release versie van Ubuntu"

#: ../UpdateManagerText/UpdateManagerText.py:117
msgid "Downloading changelog"
msgstr "Ophalen van wijzigingenlogboek"

#: ../UpdateManager/Core/MyCache.py:108
#, python-format
msgid "Other updates (%s)"
msgstr "Overige updates (%s)"

#: ../UpdateManager/Core/MyCache.py:227
#, python-format
msgid ""
"Changes for the versions:\n"
"%s\n"
"%s\n"
"\n"
msgstr ""
"Wijzigingen voor de versies:\n"
"%s\n"
"%s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:229
msgid "This change is not coming from a source that supports changelogs."
msgstr "Deze wijziging komt niet van een bron die changelogs ondersteunt."

#: ../UpdateManager/Core/MyCache.py:237
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"Het wijzigingenlogboek bevat geen relevante wijzigingen.\n"
"\n"
"Gebruik http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"totdat de wijzigingen beschikbaar worden, of probeer het later opnieuw."

#: ../UpdateManager/Core/MyCache.py:242
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"De lijst met wijzigingen is nog niet beschikbaar.\n"
"\n"
"U kunt deze wel al lezen op "
"http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"of u kunt het later nog eens proberen."

#: ../UpdateManager/Core/MyCache.py:248
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"Kon de lijst met wijzigingen niet downloaden. \n"
"Controleer uw internetverbinding."

#: ../UpdateManager/Core/UpdateList.py:50
msgid "Failed to detect distribution"
msgstr "Bepalen van de distributie is mislukt"

#: ../UpdateManager/Core/UpdateList.py:51
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr ""
"Er heeft zich een fout ‘%s’ voorgedaan bij het nakijken van de systeemversie."

#: ../UpdateManager/Core/UpdateList.py:62
msgid "Important security updates"
msgstr "Belangrijke veiligheidsupdates"

#: ../UpdateManager/Core/UpdateList.py:63
msgid "Recommended updates"
msgstr "Aanbevolen updates"

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Proposed updates"
msgstr "Voorgestelde updates"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Backports"
msgstr "Backports"

#: ../UpdateManager/Core/UpdateList.py:66
msgid "Distribution updates"
msgstr "Distributie-updates"

#: ../UpdateManager/Core/UpdateList.py:71
msgid "Other updates"
msgstr "Andere updates"

#: ../UpdateManager/Core/UpdateList.py:80
msgid "Previous selected"
msgstr "Hiervoor geselecteerd"

#: ../data/glade/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>U moet zelf controleren of er updates zijn</big></b>\n"
"\n"
"Uw systeem controleert niet automatisch of er updates zijn. U kunt dit "
"configureren in <i>Softwarebronnen</i> op het tabblad <i>Internet-"
"updates</i>."

#: ../data/glade/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>Niet alle updates kunnen worden geïnstalleerd</b></big>"

#: ../data/glade/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>Werkt op accu</b></big>"

#: ../data/glade/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<big><b>Update-beheer opstarten</b></big>"

#: ../data/glade/UpdateManager.ui.h:7
msgid "Changes"
msgstr "Wijzigingen"

#: ../data/glade/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "_Controleren"

#: ../data/glade/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "_Doorgaan"

#: ../data/glade/UpdateManager.ui.h:10
msgid "Description"
msgstr "Omschrijving"

#: ../data/glade/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "Omschrijving van de update"

#: ../data/glade/UpdateManager.ui.h:13
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"Een gedeeltelijke upgrade uitvoeren, om zoveel mogelijk updates te "
"installeren. \n"
"\n"
"Dit kan verzoorzaakt worden door:\n"
" * Een vorige upgrade die niet werd voltooid\n"
" * Problemen met de geïnstalleerde software\n"
" * Onofficiële softwarepakketten die niet ondersteund worden door Ubuntu\n"
" * Normale wijzigingen in een vooruitgave van Ubuntu"

#: ../data/glade/UpdateManager.ui.h:20
msgid "Settings..."
msgstr "Instellingen..."

#: ../data/glade/UpdateManager.ui.h:21
msgid "Show progress of individual files"
msgstr "Voortgang tonen van de afzonderlijke pakketten"

#: ../data/glade/UpdateManager.ui.h:23
msgid "U_pgrade"
msgstr "U_pgraden"

#: ../data/glade/UpdateManager.ui.h:24 ../data/update-manager.desktop.in.h:3
#: ../data/update-manager-hildon.desktop.in.h:3
msgid "Update Manager"
msgstr "Updatebeheer"

#: ../data/glade/UpdateManager.ui.h:25
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr ""
"Uw systeem werkt momenteel op een accu. Weet u zeker dat u door wilt gaan?"

#: ../data/glade/UpdateManager.ui.h:26
msgid "_Hide this information in the future"
msgstr "_Deze informatie in het vervolg niet meer tonen"

#: ../data/glade/UpdateManager.ui.h:27
msgid "_Install Updates"
msgstr "Up_dates installeren"

#: ../data/glade/UpdateManager.ui.h:28
msgid "_Partial Upgrade"
msgstr "_Gedeeltelijke upgrade"

#: ../data/glade/UpdateManager.ui.h:29
msgid "_Upgrade"
msgstr "_Upgraden"

#: ../data/glade/UpdateManager.ui.h:30
msgid "updates"
msgstr "updates"

#: ../data/update-manager.desktop.in.h:1
#: ../data/update-manager-hildon.desktop.in.h:1
msgid "Show and install available updates"
msgstr "Beschikbare updates tonen en installeren"

#: ../data/update-manager.desktop.in.h:2
#: ../data/update-manager-hildon.desktop.in.h:2
msgid "Software Updates"
msgstr "Software-updates"

#: ../data/update-manager.schemas.in.h:1
msgid "*deprecated* Check for new distribution releases"
msgstr "*verouderd* Controleren op nieuwe distributieversies"

#: ../data/update-manager.schemas.in.h:2
msgid "Auto close the install window after successful install"
msgstr "Installeervenster automatisch sluiten na voltooiing"

#: ../data/update-manager.schemas.in.h:3
msgid "First run welcome message"
msgstr "Welkomstbericht bij eerste start"

#: ../data/update-manager.schemas.in.h:4
msgid ""
"If automatic checking for updates is disabled, you have to reload the "
"channel list manually. This option allows to hide the reminder shown in this "
"case."
msgstr ""
"Indien automatisch controleren voor updates uitgeschakeld is, moet u de "
"kanalenlijst zelf herladen. Met deze optie kunt u er voor zorgen dat in dat "
"soort gevallen geen herinnering gegeven wordt."

#: ../data/update-manager.schemas.in.h:5
msgid "If this key is set a first run welcome message will be presented."
msgstr ""
"als deze optie wordt aangezet zal er een welkomstbericht gepresenteerd "
"worden."

#: ../data/update-manager.schemas.in.h:6
msgid ""
"If this key is set the install window will be automatically closed on "
"successful installation."
msgstr ""
"Bij aanvinken wordt het installeervenster automatisch gesloten na een "
"succesvolle installatie"

#: ../data/update-manager.schemas.in.h:7
msgid ""
"If this key is set the main update list window will show version information "
"(from version to version)."
msgstr ""
"Als dit aangevinkt is, zal in de lijst met updates versie-informatie getoond "
"worden (de wijzigingen tussen de oude versie en de nieuwe)."

#: ../data/update-manager.schemas.in.h:8
msgid "Remind to reload the channel list"
msgstr "Herinneren om de kanaallijst te herladen"

#: ../data/update-manager.schemas.in.h:9
msgid "Show details of an update"
msgstr "Details van een update tonen"

#: ../data/update-manager.schemas.in.h:10
msgid "Show version in update list"
msgstr "Versies tonen in de lijst met updates."

#: ../data/update-manager.schemas.in.h:11
msgid "Stores the size of the update-manager dialog"
msgstr "Onthoudt de grootte van het dialoogvenster van de update-manager"

#: ../data/update-manager.schemas.in.h:12
msgid ""
"Stores the state of the expander that contains the list of changes and the "
"description"
msgstr ""
"Onthoudt de toestand van het vensterdeel dat de lijst met wijzigingen en de "
"omschrijvingen bevat."

#: ../data/update-manager.schemas.in.h:13
msgid "The window size"
msgstr "De venstergrootte"

#: ../data/update-manager.schemas.in.h:14
msgid ""
"This key is deprecated in favor of the file /etc/update-manager/release-"
"upgrades Check automatically if a new version of the current distribution is "
"available and offer to upgrade (if possible)."
msgstr ""
"Deze sleutel is verouderd. Gebruik nu bij voorkeur het bestand /etc/update-"
"manager/release-upgrades.\r\n"
"Automatisch controleren of er een nieuwe versie van de huidige distributie "
"beschikbaar is en aanbieden om te upgraden (indien mogelijk)."

#: ../update-manager:56
msgid "Show version and exit"
msgstr "Softwareversie tonen en afsluiten"

#: ../update-manager:59
msgid "Directory that contains the data files"
msgstr "Map die databestanden bevat"

#: ../update-manager:62
msgid "Check if a new Ubuntu release is available"
msgstr "Controleren of er een nieuwe versie van Ubuntu beschikbaar is"

#: ../update-manager:65 ../do-release-upgrade:23 ../check-new-release:24
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Controleer of upgraden naar de nieuwste ontwikkelaarsversie mogelijk is"

#: ../update-manager:69
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "Upgrade uitvoeren met de meest recente versie van de upgrader"

#: ../update-manager:76
msgid "Do not focus on map when starting"
msgstr "Geen focus op de map leggen tijdens opstarten"

#: ../update-manager:79
msgid "Try to run a dist-upgrade"
msgstr "Probeer om een dist-upgrade uit te voeren"

#: ../update-manager:83 ../do-release-upgrade:39
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Upgrade testen met een sandbox-aufs-overlay"

#: ../update-manager:103
msgid "Running partial upgrade"
msgstr "Gedeeltelijke upgrade uitvoeren"

#: ../do-release-upgrade:27 ../check-new-release:28
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Probeer te upgraden naar de laatste versie door gebruik te maken van de "
"upgrade-software van $distro-proposed"

#: ../do-release-upgrade:31
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Uitvoeren in een speciale upgrade-modus\n"
"Momenteel worden 'desktop' en 'server' ondersteund voor upgrades van "
"respectievelijk een desktop-systeem en een server-systeem."

#: ../do-release-upgrade:37
msgid "Run the specified frontend"
msgstr "De opgegeven 'frontend' uitvoeren"

#: ../do-release-upgrade:43
msgid "Checking for a new ubuntu release"
msgstr "Controleren op nieuwe ubuntu versie"

#: ../do-release-upgrade:50
msgid "No new release found"
msgstr "Geen nieuwe release gevonden"

#: ../check-new-release:32
#, c-format
msgid ""
"Do not output anything, just return %s if there is a new release and %s if "
"there is none."
msgstr ""
"Geef geen uitvoer, geef %s terug als er een nieuwe uitgave is, en %s als die "
"er niet is."

#: ../check-new-release:46
#, c-format
msgid "New release '%s' available."
msgstr "Nieuwe versie '%s' beschikbaar."

#: ../check-new-release:47
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Voer 'do-release-upgrade' uit om naar de nieuwe versie te upgraden."

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "Niet-geïmplementeerde methode: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "Bestand op schijf"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb-pakket"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "Ontbrekend pakket installeren"

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "Pakket %s zou geïnstalleerd moeten worden."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i verouderde bestandsvermeldingen in statusbestand"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "Verouderde vermeldingen in dpkg-status"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "Verouderde vermeldingen in dpkg-status"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"Als kdelibs4-dev geïnstalleerd is, moet kdelibs5-dev ook geïnstalleerd "
"worden om te kunnen upgraden. Zie bug #279621 op bugs.launchpad.net voor "
"details."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s moet aangevinkt worden als handmatig geïnstalleerd."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr ""
"Lilo wordt verwijderd omdat grub ook geïnstalleerd is. (Zie bug #33140004 "
"voor meer informatie.)"

#~ msgid "Check if a new distribution release is available"
#~ msgstr "Controleren of een nieuwe versie van de distributie beschikbaar is"

#~ msgid "Upgrade to the latest version of Ubuntu"
#~ msgstr "Upgrade naar de nieuwste versie van Ubuntu"

#~ msgid "Check the software channels for new updates"
#~ msgstr "Controleer de softwarekanalen op nieuwe updates"

#, python-format
#~ msgid "<b>New distribution release '%s' is available</b>"
#~ msgstr "<b>De nieuwe distributie-versie ‘%s’ is beschikbaar</b>"

#~ msgid "Your distribution is not supported anymore"
#~ msgstr "Uw distributie wordt niet langer ondersteund"

#~ msgid "Verfication failed"
#~ msgstr "Verificatie is mislukt"

#~ msgid ""
#~ "This computer is currently using the AMD 'fglrx' graphics driver. No version "
#~ "of this driver is available that works with your hardware in Ubuntu 9.04.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Deze computer gebruikt op dit moment het grafisch stuurprogramma 'fglrx' van "
#~ "AMD. Er is geen versie van dit stuurprogramma beschikbaar die werkt met uw "
#~ "hardware in Ubuntu 9.04.\n"
#~ "\n"
#~ "Wilt u verder gaan?"

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 8.10.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Deze computer gebruikt momenteel het grafisch stuurprogramma ‘nvidia’ van "
#~ "NVIDIA. Van dit stuurprogramma is helaas geen versie beschikbaar die "
#~ "functioneert in Ubuntu 8.10.\n"
#~ "\n"
#~ "Weet u zeker dat u wilt doorgaan?"

#, python-format
#~ msgid ""
#~ "The upgrade will continue but the '%s' package may be in a not working "
#~ "state. Please consider submitting a bug report about it."
#~ msgstr ""
#~ "De upgrade zal verder gaan, maar het pakket '%s' werkt mogelijk niet meer. "
#~ "Gelieve een foutenrapport in te dienen."

#~ msgid ""
#~ "A unresolvable problem occurred while calculating the upgrade.\n"
#~ "\n"
#~ "Please report this bug against the 'update-manager' package and include the "
#~ "files in /var/log/dist-upgrade/ in the bugreport."
#~ msgstr ""
#~ "Tijdens het berekenen van de upgrade ontstond er een onoplosbaar probleem.\n"
#~ "\n"
#~ "Meld deze fout voor het pakket ‘update-manager’ in een foutrapportage en "
#~ "voeg daarbij de bestanden die zich in /var/log/dist-upgrade/ bevinden."
